docs/troubleshooting.md

```diff
@@ -0,0 +1,117 @@
+    .NET MAUI App                        maui              [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/Windows/Tizen
+    .NET MAUI Blazor App                 maui-blazor       [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/Windows/Tizen/Blazor
+    .NET MAUI Class Library              mauilib           [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/Windows/Tizen
+    .NET MAUI ContentPage (C#)           maui-page-csharp  [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/WinUI/Tizen/Xaml/Code
+    .NET MAUI ContentPage (XAML)         maui-page-xaml    [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/WinUI/Tizen/Xaml/Code
+    .NET MAUI ContentView (C#)           maui-view-csharp  [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/WinUI/Tizen/Xaml/Code
+    .NET MAUI ContentView (XAML)         maui-view-xaml    [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/WinUI/Tizen/Xaml/Code
+    .NET MAUI ResourceDictionary (XAML)  maui-dict-xaml    [C#]      MAUI/Android/iOS/macOS/Mac Catalyst/WinUI/Xaml/Code
+    ```
+
+01. Next, create a new .NET MAUI project with the `dotnet new` command, using either `maui` or `maui-blazor` as the project type:
+
+    ```dotnetcli
+    dotnet new maui
+    ```
+
+01. Open the project in Visual Studio.
+
+## ERROR Platform version is not present.
+
+Visual Studio may not be resolving the required workloads if you try to compile a project and receive an error similar to the following:
+
+```
+Platform version is not present for one or more target frameworks, even though they have specified a platform: net6.0-android, net6.0-ios, net6.0-maccatalyst
+```
+
+This may be caused by having the .NET 7 preview installed. To see if .NET 7 is being resolved as your current version of .NET, perform the following steps:
+
+01. Open a terminal.
+01. Run the `dotnet --version` command.
+
+    If the result shows a `7.0.XXX` value, you're running in the context of .NET 7. Use one of the fixes below.
+
+### Fix (1)
+
+Uninstall the .NET 7 preview.
+
+### Fix (2)
+
+Use a `global.json` config file in the same folder as your current project. This config file can force the context of .NET 6, allowing you to keep a .NET 7 preview version installed:
+
+01. Close Visual Studio.
+01. Open a terminal and navigate to the folder where your project is located.
+01. Run the following command: `dotnet new globaljson --sdk-version 6.0.300`
+01. Reopen the project in Visual Studio.
+
+## The WINDOWS `#if` directive is broken
+
+The `WINDOWS` definition doesn't resolve correctly in the latest release of .NET MAUI. To work around this issue, add the following entry to the `<PropertyGroup>` element of your project file.
+
+```
+<DefineConstants Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">$(DefineConstants);WINDOWS</DefineConstants>
+```
+
+The definitions that identify a specific version of Windows will still be missing.
+
+## ERROR Type or namespace 'Default' does not exist
```

Comment:
Personally I don't think this is required here. It's duplication of content you've already put into the Contacts doc, and it's one of many such problems.

Reply:
I think it's good to keep. This way it bubbles up in search easier. The specific warning in the contacts article won't be unless we promote it to its own section.